appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/__tests__/lpa-questionnaire.test.js

```diff
@@ -55,7 +55,7 @@ describe('LPA Questionnaire review', () => {
 			const element = parseHtml(response.text);
 
 			expect(element.innerHTML).toMatchSnapshot();
-		});
+		}, 10000);
```

Comment:
What's the reason for overriding the default timeout here? Is this a long running test?

Reply:
yes the test kept over running

Reply:
not sure why